subprojects/language-java/src/integTest/groovy/org/gradle/api/tasks/compile/JavaCompileParallelIntegrationTest.groovy

```diff
@@ -44,6 +43,8 @@ class JavaCompileParallelIntegrationTest extends AbstractIntegrationSpec {
             }
             false
         }
+        Assume.assumeTrue(result.keySet().size() >= 2)
```

Comment:
```suggestion
        Assume.assumeTrue(result.size() >= 2)
```

Reply:
IMHO this is worse than the annotation. Isn't it just hiding the fact, by using a junit construct? Shouldn't this be a `TestCondition` as well?